### DISCUSSION TITLE
Printing

### DISCUSSION THREAD

**Discussion by bombardy** [Serg-Norseman/GEDKeeper#435] [Ideas]
I've just started using this excellent program that allows me to hold unlimited generations. My records go back to 1205. I had it visually in Autocad but that proved a problem sharing.
At the moment I have just one issue and that's with multiple page printing.
When I highlight my son on the opening list of individuals and click on Pedigree / Ancestors Tree. 
Then click on a couple of yellow expansion crosses so the view enlarges beyond the size of my computer scrren. 
Then click preview. 
Then if I click the two page or four page icon it just strinks the preview to the left side of the screen rather than splitting the preview into multiple views and the page number at the top right just stays at one.
Help would be appreciated.
Thanks, Andrew

**Comment by Serg-Norseman** [comment]
Please excuse me: currently I can't improve the code for previewing and printing tree diagrams for multipage mode :(

I can only recommend that you save the diagram to a vector file and print from another tool that has the right features.

However, an issue for this question can be created. Perhaps I will be able to make the necessary changes over time.

**Reply by bombardy** [reply]
Thank you Sergey,
Do you have any suggestions as to what program might print a saved vector file?
Andrew

**Comment by Serg-Norseman** [comment]
Try [Inkscape](https://inkscape.org/)

Sergey

**Reply by bombardy** [reply]
Excellent. Thank you
Is there a downloadable version the GEDkeeper help file. A PDF that I can search?

**Comment by Serg-Norseman** [comment]
The program's help files have not been updated for many years. There is not enough time, effort and discipline to update in time with changes and improvements.

Help comes only with the program, as is. As html files; pdf assembly does not exist.

**Reply by bombardy** [reply]
Well Sergey it's a very good program and these requests are only minor. It suits me better than any other program on the internet including those that charge a continual subscrition. 
Thank you

**Comment by ghost** [comment]
Nasty solution for now:
1. Take a screenshot for each
2. Print them

This software could have "export entire tree to multiple PNG file" or "... to PDF file"

**Reply by Serg-Norseman** [reply]
Printing shops and good software split large SVG/PNG/PDF files into sections based on sheet size. When I went to the printing shop, they did a great job.
